contribute/breaking-changes-guide/breaking-changes-guide.md

```diff
@@ -16,41 +16,40 @@ This guide tries to help you identify and communicate breaking changes introduce
 
 ## What are our public APIs?
 
-The Grafana frontend codebase is exposing functionality through NPM packages to make plugin development easier and faster.
+The Grafana frontend codebase exposes functionality through NPM packages to make plugin development easier and faster.
 These packages live in the `/packages` folder and contain packages like:
 
 - `@grafana/data`
 - `@grafana/runtime`
 - `@grafana/ui`
-- etc. ([They can be viewed here](https://github.com/grafana/grafana/tree/main/packages)
+- [(more packages...)](https://github.com/grafana/grafana/tree/main/packages)
 
 Any change that causes dependent software to behave differently is considered to be breaking.
 
 ## What is Levitate?
 
-[`@grafana/levitate`](https://github.com/grafana/levitate) is a tool created by Grafana that can show breaking changes between two versions of a **TypeScript** package or a source file.
+[`@grafana/levitate`](https://github.com/grafana/levitate) is a tool created by Grafana that can show breaking changes between two versions of a TypeScript package or a source file.
 
-It can list exported members of an NPM package or imports used by an NPM package,
-**but we are mainly using it for comparing different versions of the same package to see changes in the exported members.**
+Levitate can list exported members of an NPM package or imports used by an NPM package, _but it is most commonly used for comparing different versions of the same package to see changes in the exported members._
 
 A Github workflow runs against every pull request and comments a hint in case there are
 possible breaking changes. It also adds the `"breaking change"` label to the pull request.
 
-## How does the CI workflow look like?
+## What does the CI workflow look like?
 
 <img src="./breaking-changes-workflow.png" alt="CI workflow" width="700" />
 
-## I received a comment on my PR, what does it mean?
+## What do comments on my PR mean?
 
 ![Levitate comment](./breaking-changes-comment-screenshot.png)
 
 Receiving a comment like the one above does not necessarily mean that you actually introduced breaking
```

Comment:
We should avoid "above" for directions

https://developers.google.com/style/word-list#above